src/classes/modules/java.base/java/lang/Class.java

```diff
@@ -60,6 +60,9 @@
   
   private String name;
 
+  // set by VM
```

Comment:
Could you please clarify this comment?
Does JPF's VM or the host VM try to set this field?
There is currently no accessor method, and as the field is now, it seems jpf-core would not use it at this point. However, without an accessor, the host VM could not set this field (except by reflection, which can access private data).

Reply:
Based on the JDK source there is no accessor `setModule`. But through remote debugging, I was able to confirm that the java.lang.Class#getModule method in our Class model returns the correct (non-null) module for that class.